console/console-init/ui/src/containers/AdapterListContainer/AdapterListContainer.tsx

```diff
@@ -14,24 +14,43 @@ export const AdapterListContainer: React.FC<{ id: string }> = ({ id }) => {
   const { projectname } = useParams();
 
   const queryResolver = `
-  iotProjects {
+  objects {
+    ... on IoTProject_iot_enmasse_io_v1alpha1 {
     endpoints{
       name
       url
       host
       port
       tls
     }
+  }
   }`;
 
   const { data } = useQuery<IIoTProjectsResponse>(
     RETURN_IOT_PROJECTS({ projectName: projectname }, queryResolver)
   );
 
   const { allProjects } = data || {
-    allProjects: { iotProjects: [] }
+    allProjects: { objects: [] }
   };
-  const adapters: IAdapter[] = [];
+  let adapters: IAdapter[] = [];
+  if (
+    allProjects &&
+    allProjects.objects?.length > 0 &&
+    allProjects.objects[0].endpoints
+  ) {
+    const { endpoints } = allProjects.objects[0];
+    let adapterlist = endpoints.map(endpoint => {
```

Comment:
I don't think so we need looping over there. You can pass endpoints itself and AdapterList component will handle it.
 return <AdapterList id={id} adapters={endpoints} />;